presto-parquet/src/test/java/com/facebook/presto/parquet/batchreader/decoders/TestParquetUtils.java

```diff
@@ -144,6 +144,16 @@ public static byte[] generatePlainValuesPage(int valueCount, int valueSizeBits,
                 }
                 break;
             }
+            case 128:
+                for (int i = 0; i < valueCount; i++) {
+                    long value = random.nextLong();
+                    writer.writeLong(value);
+                    addedValues.add(value);
+                    value = random.nextLong();
```

Comment:
no random values please; pick an arbitrary one

Reply:
The entire test class revolves around generating random values for different primitive types. I don't think we should go against the grain here. Changing this entire class to not use random values is out of scope for this PR.

Reply:
I'm not asking for the entire class to change. I'm asking that this PR not make the problem worse. Random values in unit tests are a major source of flakiness. Please don't do this.

Reply:
I am not going to change this because it requires re-writing a lots of boilerplate of code to perform the same assertions. I filed an issue for someone to fix this class in the future: https://github.com/prestodb/presto/issues/23840

Reply:
Instead of random values why not just i, -79L*i, 237L*i?